CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Main (unreleased)
 
 - Some blocks in Flow components have been merged with their parent block to make the block hierarchy smaller:
   - `prometheus.scrape > client > http_client_config` is merged into the `client` block. (@erikbaranowski)
+  - `loki.source.kubernetes > client > client> http_client_config` is merged into the `client > client` block. (@erikbaranowski)
```

Comment:
Are you sure this is right? There's two levels of client blocks?

Reply:
I've taken the first > client to represent the top level in these comments so there is another layer on this component with exactly that name.